playbooks/support/scheduling.md

```diff
@@ -107,5 +78,28 @@ To override a shift:
 
    <img width=500 src="images/opsgenie-override-2.png" />
 
-[schedule]: https://artsy.app.opsgenie.com/teams/dashboard/ee381004-a72e-42ef-a733-b350d6693c6c/main
-[artsy-cli]: https://github.com/artsy/artsy-cli
+## **Adding Engineers**
+
+Generally, new engineers should be added to the end of the schedule rotation associated with their time zone. To do this: 
+
+1. Navigate to the [On-Call Schedule](https://artsy.app.opsgenie.com/teams/dashboard/ee381004-a72e-42ef-a733-b350d6693c6c/main) and edit the engineer's respective rotation.
+
+   <img width=500 src="images/opsgenie-addengs-1.png" />
+   <img width=500 src="images/opsgenie-addengs-2.png" />
+
+2. Add the engineer to the end of the participants list. Their position in the rotation will match their position in the participants list on the rotation edit page, for example:
+
+Considerations when adding a new engineer:
+
+- You should aim to avoid pairings with two new engineers and you can place the engineer earlier in the rotation to avoid this.
+- **NOTE: If you add engineers anywhere other than the end of the rotation, the schedule of the engineers who have not completed their shifts will be modified, as the order of the participants matters. You should therefore update the affected engineers to avoid scheduling conflicts.**
```

Comment:
Can we have some guidelines around this?

Like-- How far in advance should we assume the schedule is "set"? If the protocol is to inform engineers whose schedules may have changed, can we provide a sample slack message?

Reply:
Since we won't add new engineers within 60 days of the rotation completion, we can assume the schedule will no longer shift (other than individual swaps) at this point.

And will add a sample slack message! Good call.